### PR TITLE
chore: Linting

### PR DESCRIPTION
**Pull request by jfaracik**
_Finally_ LITE Linter is added to lite-internal-frontend. 

**Changes**

* Adds LITE Linter v1.0 to LITE!
* Converts title H2s to H1s
* Converted spaces to tabs
* Corrects spacing between blocks
* Moved Add user, Edit user, Add flag, Edit flag, Add team, edit team form over to SingleFormView
* Some misc changes (moving Javascript to Javascript blocks)

Cat gif as payment for reviewing this monstrosity:
https://i.imgur.com/7nLm7si.mp4

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -26,18 +26,18 @@
         },
         "boto3": {
             "hashes": [
-                "sha256:25e6af364243673bacfe4bd422582f4a891e02ae433e266b03de6a6bfdd63d51",
-                "sha256:4ba878e6ea38d3821b571e29018e00f8a9ddab2678dd59433b1ecda331cf9aa6"
+                "sha256:5cfe1e609af34e1641402e016047e2d12452992b52dccd3e5caea0990f225dc3",
+                "sha256:6a45d2b777821b9b4b69656f628b498b86f6906c6643eca7759c660b26eb541e"
             ],
             "index": "pypi",
-            "version": "==1.11.11"
```
Comment (phlptt) [inline Pipfile.lock]: can you deploy it, it looks like the semantic versioning isnt working as i'd expect

boto3 = "~=1.11" is what's in the pipfile

Comment (phlptt) [reply Pipfile.lock]: or fix the semantic versioning

File: templates/case/views/application-information-board.html [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
-                Reason(s) for query
-            </dt>
-            <dd class="govuk-summary-list__value">
-                {{ case.application.reasoning }}
-            </dd>
-        </div>
-    {% endif %}
+	<div class="govuk-summary-list__row">
+		<dt class="govuk-summary-list__key">
+			Activity
```
Comment (stephanieboyd) [inline templates/case/views/application-information-board.html]: You touched them :O

Comment (jfaracik) [reply templates/case/views/application-information-board.html]: I'll happily add strings BUT I won't do it for files that are going to be deleted/changed when the big ol' case UI refactor story comes in, happy? :)

File: Pipfile.lock (hunk 1 above)
Comment (jfaracik) [reply Pipfile.lock]: `~=1.11` would include version 1.12 and any minor update, but not 2.0. 

major.minor.micro

**Pull request merged**